src/shape.rs

```diff
@@ -137,9 +137,16 @@ pub trait Shape {
     /// [winding number]: https://mathworld.wolfram.com/ContourWindingNumber.html
     fn winding(&self, pt: Point) -> i32;
 
-    /// Returns `true` if the [`Point`] is inside this shape.
+    /// Returns `true` if the [`Point`] is inside this shape using the non-zero
+    /// winding rule.
     ///
-    /// This is only meaningful for closed shapes.
+    /// This is only meaningful for closed shapes. Some shapes may have specialized
+    /// implementations of this function or of [`winding`] determination.
```

Comment:
All shapes must implement `Shape::winding`, so perhaps this can just say ``Some shapes may have specialized implementations of this function rather than calculating [`winding`].``

Reply:
I don’t think any actually override `contains` instead overriding `winding`.

Reply:
I should edit it to say that by default it is the non-zero winding rule since an override might not be the same.